src/store/ducks/structures.ts

```diff
@@ -0,0 +1,134 @@
+import { keyBy, values } from 'lodash';
+import { AnyAction, Store } from 'redux';
+import SeamlessImmutable from 'seamless-immutable';
+import { FeatureCollection, GeoJSON, UpdateType, wrapFeatureCollection } from '../../helpers/utils';
+
+/** the reducer name */
+export const reducerName = 'structures';
+
+/** Interface for structure.geojson.properties for structure
+ *  as received from the fetch request / superset
+ */
+export interface InitialProperties {
```

Comment:
If we renamed this to `Structure` we would not need line 40 below at all.  What's the reason for it being this way?